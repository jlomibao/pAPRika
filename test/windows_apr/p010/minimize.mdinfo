

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6890E-01     1.5997E+00     H4         17

 BOND    =        1.1104  ANGLE   =      107.7353  DIHED      =       51.4549
 VDWAALS =      -46.5313  EEL     =     1337.4835  EGB        =     -135.5775
 1-4 VDW =        5.2124  1-4 EEL =    -2132.2039  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================
