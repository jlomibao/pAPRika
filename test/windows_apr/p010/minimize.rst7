defa
  125
  -0.5217225   4.9759547   1.8352663   1.0157702   4.9108285   1.8352068
  -0.8854964   4.2790898   0.6084356  -0.8854083   4.2789259   3.0620277
  -0.9016843   6.0040954   1.8353475   1.4813224   5.9031517   1.8352558
   1.3195550   4.1855637   0.6086211   1.3196391   4.1853890   3.0616689
   2.6868515   4.0833250   0.1137486   2.6869520   4.0830881   3.5564659
   0.2013732   3.8585050   3.7830652   0.2012348   3.8587592  -0.1127205
   0.1803004   3.3490141   4.8904593   0.1800731   3.3493053  -1.2201559
   2.6632924   4.0469581  -0.9814997   3.2459753   4.9656298   0.4459569
   2.6634020   4.0464931   4.6517069   3.2460521   4.9654677   3.2244162
  -4.6359435   1.9609700   1.8353369  -3.8101033   3.2595333   1.8352982
  -4.2146126   1.2971689   0.6085833  -4.2146283   1.2971765   3.0621136
  -5.7162254   2.1465373   1.8353336  -4.4362751   4.1591738   1.8353079
  -3.0303082   3.1594909   0.6085151  -3.0303197   3.1594820   3.0621265
  -2.2565789   4.2914362   0.1136141  -2.2564245   4.2913167   3.5570140
  -3.3067881   2.0275563   3.7832546  -3.3067453   2.0275283  -0.1125511
  -2.8760077   1.7540732   4.8905039  -2.8758673   1.7539681  -1.2197664
  -2.2363404   4.2525849  -0.9816153  -2.7399944   5.2175044   0.4455039
  -2.2360112   4.2523356   4.6522356  -2.7398194   5.2174566   3.2252942
  -4.0834333  -3.1098896   1.8352338  -4.7950144  -1.7453491   1.8352887
  -3.2979693  -3.0769580   0.6084259  -3.2978165  -3.0769759   3.0619466
  -4.7843304  -3.9526158   1.8352684  -5.8872212  -1.8376991   1.8353254
  -4.3184363  -1.1200686   0.6084864  -4.3183351  -1.1201054   3.0620713
  -4.9114153   0.1162772   0.1138034  -4.9113031   0.1161973   3.5568812
  -3.4762823  -1.9255648   3.7831158  -3.4764520  -1.9255023  -0.1126545
  -3.0239091  -1.6892740   4.8903149  -3.0240735  -1.6891170  -1.2198561
  -4.8677937   0.1144728  -0.9814319  -5.9550504   0.1610931   0.4458592
  -4.8675437   0.1143117   4.6521109  -5.9549807   0.1609831   3.2249550
   0.5850545  -5.1654506   1.8348872  -0.9524993  -5.1002084   1.8349920
   0.9488663  -4.4684484   0.6081216   0.9490162  -4.4685792   3.0616815
   0.9649533  -6.1936146   1.8348020  -1.4181176  -6.0925006   1.8349799
  -1.2561362  -4.3749111   0.6082599  -1.2559789  -4.3749968   3.0618236
  -2.6234385  -4.2707667   0.1137605  -2.6232069  -4.2707801   3.5565172
  -0.1375634  -4.0480096   3.7828884  -0.1378123  -4.0478071  -0.1128920
  -0.1162429  -3.5381616   4.8901223  -0.1166320  -3.5377449  -1.2200547
  -2.6002409  -4.2341101  -0.9814859  -3.1837629  -5.1522639   0.4460900
  -2.5998151  -4.2340597   4.6517574  -3.1836085  -5.1522798   3.2243244
   4.6985415  -2.1490946   1.8347377   3.8734057  -3.4481104   1.8347418
   4.2767865  -1.4854334   0.6080489   4.2769042  -1.4855136   3.0615070
   5.7789214  -2.3340903   1.8346784   4.5000261  -4.3474385   1.8346866
   3.0934783  -3.3484038   0.6080041   3.0935935  -3.3484706   3.0615584
   2.3199516  -4.4805802   0.1132969   2.3201608  -4.4806798   3.5563169
   3.3694918  -2.2163980   3.7826814   3.3693019  -2.2162817  -0.1130671
   2.9386532  -1.9431662   4.8899708   2.9382783  -1.9429349  -1.2202810
   2.2996564  -4.4418592  -0.9819360   2.8035426  -5.4065211   0.4452864
   2.2999951  -4.4419764   4.6515529   2.8037363  -5.4065942   3.2242312
   4.1487810   2.9227308   1.8349628   4.8584063   1.5570328   1.8348523
   3.3640743   2.8906147   0.6075590   3.3641936   2.8904842   3.0624453
   4.8510006   3.7643553   1.8349681   5.9507469   1.6477864   1.8347857
   4.3808109   0.9320351   0.6082443   4.3809575   0.9319370   3.0614727
   4.9732463  -0.3044118   0.1130609   4.9734135  -0.3045429   3.5565141
   3.5394393   1.7379239   3.7825256   3.5392173   1.7381042  -0.1126309
   3.0855537   1.5012122   4.8890152   3.0851247   1.5014571  -1.2190749
   4.9290056  -0.3024326  -0.9821492   6.0170946  -0.3493598   0.4444281
   4.9292275  -0.3026376   4.6517266   6.0172362  -0.3494790   3.2250650
   0.0001401   0.0029556  10.5429093  -0.9139850  -0.5759748  10.3628854
   0.8629824  -0.6499371  10.3628323   0.0341563   0.8208054   9.8127299
   0.0233953   0.5607774  11.9791252   0.9308086   1.1612125  12.1179622
  -0.0232016  -0.5608517  13.0386263  -0.8310273   1.2344719  12.1180300
  -0.9305068  -1.1612883  12.8990987   0.8311923  -1.2344560  12.8991174
  -0.0000915  -0.0039913  14.4754745  -0.8629993   0.6488631  14.6553939
  -0.0341346  -0.8224214  15.2050010   0.9140130   0.5749916  14.6554424
  -0.0001125   0.0002824  -6.0041330  -0.0001997   0.0000632  -9.0000703
   0.0001654   2.2000035 -11.2000237
