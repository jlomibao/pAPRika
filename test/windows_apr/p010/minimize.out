
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:28:16

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p010
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   837186 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9700E+02     1.0076E+01     2.4803E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.9540  EEL     =     1326.5697  EGB        =     -129.3152
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -796.9974
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1131E+02     4.6899E-01     1.5997E+00     H4         17

 BOND    =        1.1086  ANGLE   =      107.7344  DIHED      =       51.4571
 VDWAALS =      -46.5268  EEL     =     1337.4307  EGB        =     -135.5642
 1-4 VDW =        5.2100  1-4 EEL =    -2132.1660  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1131E+02     4.6898E-01     1.5997E+00     H4         17

 BOND    =        1.1088  ANGLE   =      107.7345  DIHED      =       51.4568
 VDWAALS =      -46.5273  EEL     =     1337.4363  EGB        =     -135.5656
 1-4 VDW =        5.2102  1-4 EEL =    -2132.1700  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1131E+02     4.6898E-01     1.5997E+00     H4         17

 BOND    =        1.1090  ANGLE   =      107.7346  DIHED      =       51.4566
 VDWAALS =      -46.5277  EEL     =     1337.4415  EGB        =     -135.5670
 1-4 VDW =        5.2105  1-4 EEL =    -2132.1736  RESTRAINT  =        0.0098
 EAMBER  =     -811.3161
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1131E+02     4.6896E-01     1.5997E+00     H4         17

 BOND    =        1.1092  ANGLE   =      107.7347  DIHED      =       51.4564
 VDWAALS =      -46.5283  EEL     =     1337.4479  EGB        =     -135.5686
 1-4 VDW =        5.2108  1-4 EEL =    -2132.1782  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1131E+02     4.6896E-01     1.5997E+00     H4         17

 BOND    =        1.1094  ANGLE   =      107.7348  DIHED      =       51.4561
 VDWAALS =      -46.5288  EEL     =     1337.4538  EGB        =     -135.5701
 1-4 VDW =        5.2110  1-4 EEL =    -2132.1824  RESTRAINT  =        0.0098
 EAMBER  =     -811.3161
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1131E+02     4.6894E-01     1.5997E+00     H4         17

 BOND    =        1.1096  ANGLE   =      107.7349  DIHED      =       51.4559
 VDWAALS =      -46.5293  EEL     =     1337.4597  EGB        =     -135.5715
 1-4 VDW =        5.2113  1-4 EEL =    -2132.1868  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1131E+02     4.6894E-01     1.5997E+00     H4         17

 BOND    =        1.1098  ANGLE   =      107.7350  DIHED      =       51.4556
 VDWAALS =      -46.5298  EEL     =     1337.4655  EGB        =     -135.5730
 1-4 VDW =        5.2116  1-4 EEL =    -2132.1908  RESTRAINT  =        0.0098
 EAMBER  =     -811.3161
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1131E+02     4.6893E-01     1.5997E+00     H4         17

 BOND    =        1.1100  ANGLE   =      107.7351  DIHED      =       51.4554
 VDWAALS =      -46.5303  EEL     =     1337.4715  EGB        =     -135.5745
 1-4 VDW =        5.2119  1-4 EEL =    -2132.1950  RESTRAINT  =        0.0098
 EAMBER  =     -811.3161
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1131E+02     4.6892E-01     1.5997E+00     H4         17

 BOND    =        1.1102  ANGLE   =      107.7352  DIHED      =       51.4551
 VDWAALS =      -46.5308  EEL     =     1337.4777  EGB        =     -135.5761
 1-4 VDW =        5.2121  1-4 EEL =    -2132.1996  RESTRAINT  =        0.0098
 EAMBER  =     -811.3161
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6890E-01     1.5997E+00     H4         17

 BOND    =        1.1104  ANGLE   =      107.7353  DIHED      =       51.4549
 VDWAALS =      -46.5313  EEL     =     1337.4835  EGB        =     -135.5775
 1-4 VDW =        5.2124  1-4 EEL =    -2132.2039  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6890E-01     1.5997E+00     H4         17

 BOND    =        1.1104  ANGLE   =      107.7353  DIHED      =       51.4549
 VDWAALS =      -46.5313  EEL     =     1337.4835  EGB        =     -135.5775
 1-4 VDW =        5.2124  1-4 EEL =    -2132.2039  RESTRAINT  =        0.0098
 EAMBER  =     -811.3162
 NMR restraints: Bond =    0.009   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.306    9.302    0.004    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.400  147.398    0.002    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.473   49.483    0.010    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.012   62.003    0.009    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.194 -146.178    0.015    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.551  -72.548    0.003    0.000 t
  DUM  DM1    3 --  C    BUT    2:   16.547   16.588    0.041    0.008 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.995  180.000    0.005    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.889  180.000    0.111    0.000 a
                                       Total distance penalty:      0.009
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.472
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.17 (21.61% of Gen B)
|                Calc gb diag               3.89 (38.78% of Gen B)
|                Calc gb off-diag           3.97 (39.59% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.02 (100.0% of Nonbo)
|          Nonbond force             10.02 (90.40% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.29% of Force)
|          Other                      0.03 ( 0.31% of Force)
|       Force time                11.09 (100.0% of Runmd)
|    Runmd Time                11.09 (99.80% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.11 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:28:16.833  on 04/10/2019
|           Setup done at 18:28:16.838  on 04/10/2019
|           Run   done at 18:28:27.945  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
